stablehlo/dialect/VhloAttrs.td

```diff
@@ -22,6 +22,25 @@ include "stablehlo/dialect/VhloEnums.td"
 
 include "mlir/IR/AttrTypeBase.td"
 
+//===----------------------------------------------------------------------===//
+// Attribute Versioning
+//===----------------------------------------------------------------------===//
+
+class VHLO_AttrDef<string name,
+                   string minVersion = "0.3.0",
+                   string maxVersion = "current">
```

Comment:
What do you think about removing default values for minVersion and maxVersion and explicitly specifying them everywhere? This way each entity's minimum and maximum version will be easily discoverable without reliance on code search. 

Also, for all ops, attributes and types that will be added after this PR, the minVersion default will be wrong, which might lead to confusion that we've already observed in https://github.com/openxla/stablehlo/pull/673#discussion_r1043710870. If removing default values for minVersion and maxVersion sounds like too much, perhaps let's remove the default value for minVersion?

Reply:
> If removing default values for minVersion and maxVersion sounds like too much, perhaps let's remove the default value for minVersion

I was thinking this. Lets do this in a follow up change, will be easier to review in isolation. Created  #779